MCPB_README.md

```
@@ -0,0 +1,172 @@
+- **json**: Standard JSON format
+- **jsonv2**: Enhanced JSON format (default)
+- **geojson**: GeoJSON format
+- **geocodejson**: GeocodeJSON format
+- **xml**: XML format
+
+## Data Source and License
+
+This extension uses the OpenStreetMap Nominatim API:
+- **Data**: © OpenStreetMap contributors
+- **License**: ODbL 1.0
+- **Copyright**: http://osm.org/copyright
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)